ODIN_II/SRC/netlist_utils.cpp

```diff
@@ -245,7 +245,7 @@ npin_t* free_npin(npin_t* to_free) {
  *  Should only be called by the parent functions,
  *  copy_input_npin & copy_output_npin
  *-----------------------------------------------------------------------*/
-static npin_t *copy_npin(npin_t *copy_pin)
```

Comment:
functions are declared static if you don't expect to use them outside the file, this way you can reuse the name more than once and it can avoid weird errors. also, it reduces code size since static function don't need to be linked.

Reply:
I need this function in the ga_adder.cpp (GA functions at the end of partial_map.cpp), so in the other ways, I can not use it there!